pages/apim/3.x/user-guide/publisher/service-discovery/service-discovery-consul.adoc

```diff
@@ -1,7 +1,132 @@
+= Consul.io Service Discovery
 :page-sidebar: apim_3_x_sidebar
 :page-permalink: apim/3.x/apim_service_discovery_consul.html
 :page-folder: apim/user-guide/publisher/service-discovery
 :page-layout: apim3x
 :page-title: Consul.io service discovery
 
-include::https://raw.githubusercontent.com/gravitee-io/gravitee-service-discovery-consul/master/README.adoc[]
+// include::https://raw.githubusercontent.com/gravitee-io/gravitee-service-discovery-consul/master/README.adoc[]
+
+== Introduction
+
+https://www.consul.io[Consul^] is a service mesh solution providing a full featured control plane with service discovery, configuration, and segmentation functionality.
+
+. *Service Discovery*: One of the main goals of service discovery is to provide a catalog of available services and to potentially associate it with a health check. Clients of Consul can register a service, such as an backend api, and other clients can use Consul to discover providers of a given service. Using either DNS or HTTP, applications can easily find the services they depend upon.
+
+. *Health Checking*: Consul clients can provide any number of health checks, either associated with a given service ("is the webserver returning 200 OK"), or with the local node ("is memory utilization below 90%"). This information is used by the service discovery components to route traffic away from unhealthy hosts.
+
+*Gravitee.io Service discovery for Consul.io allows you to bind the backend endpoints of your API to a service managed by Consul so that API request requests are always routed to the proper, healthy backend service dynamically managed by Consul.*
```

Comment:
Typo

```suggestion
*Gravitee.io Service discovery for Consul.io allows you to bind the backend endpoints of your API to a service managed by Consul so that API requests are always routed to the proper, healthy backend service dynamically managed by Consul.*
```